fxa/__main__.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import argparse
+import getpass
+import os
+import re
+import sys
+
+from fxa.errors import ClientError
+from fxa.tools.bearer import get_bearer_token
+from fxa.tools.browserid import get_browserid_assertion
+from fxa.tools.fxa import create_new_fxa_account
+
+FXA_API_URL = "https://api-accounts.stage.mozaws.net/v1"
+FXA_OAUTH_URL = "https://oauth.stage.mozaws.net/v1"
```

Comment:
These differ from the default endpoints used by the rest of the lib; it might be a good opportunity to refactor them into a set of shared constants in a single location rather than having different modules in this pacakge use different default URLs.